### PR TITLE
Migrate logs in staging/src/k8s.io/kube-aggregator/pkg/apiserver to structured logging

### PR DESCRIPTION
**Pull request by gaoguangze111**
Migrate log in staging/src/k8s.io/kube-aggregator/pkg/apiserver to
structured log.

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Migrate some logs in  staging/src/k8s.io/kube-aggregator/pkg/apiserver to structured logging.

**Which issue(s) this PR fixes**:

Fixes #
staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go
staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

Ref:
- keps/sig-instrumentation/1602-structured-logging
- Structured Logging migration instructions

### CODE REVIEW

File: staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ func (c *APIServiceRegistrationController) processNextWorkItem() bool {
 func (c *APIServiceRegistrationController) enqueueInternal(obj *v1.APIService) {
 	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
 	if err != nil {
-		klog.Errorf("Couldn't get key for object %#v: %v", obj, err)
+		klog.ErrorS(err, "Couldn't get key for object", "object", obj)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:161]: What will this emit? The existing line emits type information that would be useful for debugging.

File: staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go [hunk 2]
```diff
@@ -167,13 +167,13 @@ func (c *APIServiceRegistrationController) enqueueInternal(obj *v1.APIService) {
 
 func (c *APIServiceRegistrationController) addAPIService(obj interface{}) {
 	castObj := obj.(*v1.APIService)
-	klog.V(4).Infof("Adding %s", castObj.Name)
+	klog.V(4).InfoS("Adding", "object", castObj.Name)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:170]: This is a name, not an object

File: staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go [hunk 3]
```diff
@@ -167,13 +167,13 @@ func (c *APIServiceRegistrationController) enqueueInternal(obj *v1.APIService) {
...
 	c.enqueueInternal(castObj)
 }
 
 func (c *APIServiceRegistrationController) updateAPIService(obj, _ interface{}) {
 	castObj := obj.(*v1.APIService)
-	klog.V(4).Infof("Updating %s", castObj.Name)
+	klog.V(4).InfoS("Updating", "object", castObj.Name)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:176]: This is a name, not an object

File: staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go [hunk 4]
```diff
@@ -182,16 +182,16 @@ func (c *APIServiceRegistrationController) deleteAPIService(obj interface{}) {
...
 		}
 		castObj, ok = tombstone.Obj.(*v1.APIService)
 		if !ok {
-			klog.Errorf("Tombstone contained object that is not expected %#v", obj)
+			klog.ErrorS(nil, "Tombstone contained object that is not expected", "object",obj)
 			return
 		}
 	}
-	klog.V(4).Infof("Deleting %q", castObj.Name)
+	klog.V(4).InfoS("Deleting", "object", castObj.Name)
```
Comment (lavalamp) [inline staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:194]: This is a name, not an object

File: staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go (hunk 1 above)
Comment (ehashman) [reply staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:161]: Info on `ErrorS` from the migration guide: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes

**Review by ehashman — COMMENTED** [review]
This change will require a release note, please add one.

**Comment by shubheksha** [comment]
/retitle Migrate logs in staging/src/k8s.io/kube-aggregator/pkg/apiserver to structured logging

**Comment by yangjunmyfm192085** [comment]
need rebase @gaoguangze111, could you do it?

File: staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go (hunk 1 above)
Comment (cheftako) [reply staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:161]: Please correct me if you disagree Daniel, but I think your asking for an additional field in the structured output which is the type? The # automatically adding that information to the old logged output. https://play.golang.org/p/ULYQ34hInsK shows the difference between ```%#v``` and ```%v``` and how that additional field could be obtained.

Comment (lavalamp) [reply staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiservice_controller.go:161]: Yeah, that's what I was thinking. Although I now note that despite the generic name and `%#v`, 'obj' is actually strongly typed so it isn't actually important.

**Pull request closed without merging**